ethcore/src/engines/mod.rs

```diff
@@ -71,8 +71,9 @@ pub trait Engine : Sync + Send {
 	/// Block transformation functions, after the transactions.
 	fn on_close_block(&self, _block: &mut ExecutedBlock) {}
 
-	/// If true, generate_seal has to be implemented.
-	fn seals_internally(&self) -> bool { false }
+	/// If Some(true) this author is able to generate seals, generate_seal has to be implemented.
```

Comment:
Sure, I was thinking:
`None` => this question is not relevant to this `Enigne`
`Some(_)` => here is the answer.